src/main.c

```diff
@@ -262,8 +262,24 @@ motion_notify(XEvent *e)
 }
 
 int
-main()
+main(int argc, char *args[])
 {
+    if (argc > 1)
+    {
+        die("Usage: %s\n\n"
+            "KEYBINDS                       DESCRIPTION\n"
+            "[0]                            Reset the application state (position, scale, velocity, etc).\n"
+            "[q] or [ESC]                   Quit the application.\n"
+            "[r]                            Reload configuration.\n"
+            "[Ctrl] + [r]                   Reload the shaders (only for Developer mode)\n"
```

Comment:
I realize that this is copied from boomer's help menu, but some things are a bit off here. We dont have a developer mode as the original boomer does. Its _technically_ on by default as the shaders arent compiled into the program so theres no Ctrl+r or any dev-mode keybinds although a pr implementing this would just need to reload and recompile the shader. Other than that i agree that a help menu would be useful, so good job implementing that :D